examples/Binance/CoinMFutures.jl

```diff
@@ -53,3 +53,12 @@ Binance.CoinMFutures.income_log(
     endTime = DateTime("2023-07-23"),
     limit = 1000,
 )
+
+Binance.CoinMFutures.order_book(; symbol = "BTCUSD_PERP")
+Binance.CoinMFutures.order_book(; symbol = "BTCUSD_PERP", limit = 10)
+
+Binance.CoinMFutures.premium_index(; pair = "BTCUSD")
+Binance.CoinMFutures.premium_index(; symbol = "BTCUSD_PERP")
+
+Binance.CoinMFutures.ticker(; pair = "BTCUSD")
+Binance.CoinMFutures.ticker(; symbol = "BTCUSD_PERP")
```

Comment:
There was a duplicate of the examples here
Exactly the same examples are given in lines 31-38